facs/bloom.c

```diff
@@ -218,6 +218,7 @@ int save_bloom (char *filename, bloom * bl, char *prefix, char *target)
 {
   char *bloom_file = NULL;
   int fd;
+	int check_ftruncate = 1;
```

Comment:
It is indented, just with a tab instead of 3 spaces. How important is this? :wink: